packages/keyring-controller/src/KeyringController.test.ts

```diff
@@ -595,6 +595,51 @@ describe('KeyringController', () => {
     });
   });
 
+  describe('decryptMessage', () => {
+    it('should successfully decrypt a message with valid parameters and return the raw decryption result', async () => {
+      await withController(async ({ controller }) => {
+        const messageParams = {
+          from: '0x1234567890abcdef',
+          data: {
+            version: '1.0',
+            nonce: '123456',
+            ephemPublicKey: '0xabcdef1234567890',
+            ciphertext: '0xabcdef1234567890',
+          },
+        };
+        const expectedDecryptionResult = 'decrypted message';
+
+        const decryptMessageMock = jest
+          .fn()
+          .mockResolvedValue(expectedDecryptionResult);
+        controller.decryptMessage = decryptMessageMock;
```

Comment:
Hmm. It seems that we are replacing the method that we want to test with a mock, so we're not really testing what it does. Is there another way we can write this test?